pit30m/data/log_reader.py

```diff
@@ -75,17 +75,53 @@ def __init__(
             wgs84_pose_fname: Name of the WGS84 pose file (ie, global coords). This is usually "wgs84.npz.lz4"
             map: Map object. If not provided, will try to load it from the log root
             index_version: Version of the index to use. Currently only 0 is supported.
+            partitions: Set of partitions to load. These are used to load subset of the date (e.g., training queries).
+                defaults to None, which means that no sensor measurements are filtered.
         """
         self._log_root_uri = log_root_uri.rstrip("/")
         self._pose_fname = pose_fname
         self._wgs84_pose_fname = wgs84_pose_fname
         self._map = Map() if map is None else map
         # TODO(julieta) Semantic version this
         self._index_version = index_version
+        self.partitions = set() if partitions is None else partitions
 
     def __repr__(self) -> str:
         return f"Pit30M Log Reader: {self._log_root_uri}"
 
+    @cached_property
+    def partitions_index(self) -> np.ndarray:
```

Comment:
Should this be called `partition_mask`?

Reply:
Addressed in [0b0e9ab](https://github.com/pit30m/pit30m/pull/35/commits/0b0e9ab1589ad35c32c384b72f58b1b9504fd763)